xtask/src/codegen/aya_ebpf_bindings.rs

```diff
@@ -57,6 +57,7 @@ pub fn codegen(opts: &SysrootOptions, libbpf_dir: &Path) -> Result<(), anyhow::E
             "user_pt_regs",
             "user_regs_struct",
             "xdp_action",
+            "tcx_action_base",
```

Comment:
Per usual codegen policy can you split this commit out so we can run this through the codegen workflow.

Reply:
https://github.com/aya-rs/aya/pull/1006